controllers/machineinventory_controller.go

```diff
@@ -53,6 +53,7 @@ import (
 const adoptionTimeout = 5
 
 const LocalResetPlanPath = "/oem/reset-cloud-config.yaml"
+const LocalResetUnmanagedMarker = "/etc/rancher/.unmanaged_reset"
```

Comment:
It is unlikely that we will ever collide with Rancher /etc on this file, but it would be best to move it to `/etc/elemental` instead.
`/var/lib/elemental` would also be a good candidate, since `/etc` may be read-only on an immutable system.

Reply:
Thanks, will update. I've moved to `/var/lib/elemental` just to make sure that we can maximise coverage on immutable systems.